pkg/cmd/server/api/v1/types.go

```diff
@@ -970,8 +970,16 @@ type KubernetesMasterConfig struct {
 	ServicesNodePortRange string `json:"servicesNodePortRange"`
 	// StaticNodeNames is the list of nodes that are statically known
 	StaticNodeNames []string `json:"staticNodeNames"`
+
 	// SchedulerConfigFile points to a file that describes how to set up the scheduler. If empty, you get the default scheduling rules.
 	SchedulerConfigFile string `json:"schedulerConfigFile"`
+	// SchedulerName is the name this scheduler uses to assign pods.
```

Comment:
I think doing that is going to require fixing the upstream scheduler args,
because it includes the scheduler, so it would get added into oc.

On Wed, Jul 27, 2016 at 4:04 PM, Jordan Liggitt <notifications@github.com>
wrote:

> In pkg/cmd/server/api/v1/types.go
> <https://github.com/openshift/origin/pull/10064#discussion_r72512606>:
>
> >  	// SchedulerConfigFile points to a file that describes how to set up the scheduler. If empty, you get the default scheduling rules.
> >  	SchedulerConfigFile string `json:"schedulerConfigFile"`
> > +	// SchedulerName is the name this scheduler uses to assign pods.
>
> I'd rather have schedulerArguments, override the default scheduler file
> with the first-class field from our config, and be perfectly consistent
> with no effort
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/10064/files/b4296d1dbcfd5294ac9467d4d42ad735d1cfbc5d#r72512606>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABG_p_GrH3yWraronSVzq3cZSn9zYJ3wks5qZ7nogaJpZM4JWjvW>
> .
>